anylabeling/views/labeling/utils/opencv.py

```diff
@@ -1,9 +1,22 @@
+import cv2
+import numpy as np
 import qimage2ndarray
 from PyQt5 import QtGui
 
 
 def qt_img_to_cv_img(in_image):
-    return qimage2ndarray.rgb_view(in_image)
+    """
+    allow to load rgb,gray image to inference
```

Comment:
Thank you for your contribution. Please update the following to make it consistent with other parts:
- Please uppercase all first characters in all comments.
- Rename this function to `qt_img_to_rgb_cv_img()`.
- Comment should be: `Convert QtImage to OpenCV RGB image`.
- Update other files using this function also.